data/dataexchange/src/main/java/tech/pegasys/teku/data/SlashingProtectionImporter.java

```diff
@@ -85,11 +84,12 @@ public void initialise(final File inputFile) throws IOException {
 
     } catch (JsonMappingException e) {
       String cause = e.getCause() != null ? e.getCause().getMessage() : e.getMessage();
-      log.exit(1, "Failed to load data from " + inputFile.getName() + ". " + cause);
+      return Optional.of(String.format("Failed to load data. %s", cause));
     } catch (JsonParseException e) {
       String cause = e.getCause() != null ? e.getCause().getMessage() : e.getMessage();
-      log.exit(1, "Json does not appear valid in file " + inputFile.getName() + ". " + cause);
+      return Optional.of(String.format("Json does not appear valid. %s", cause));
```

Comment:
nit: String.format seems rather overkill for just appending data here - would be shorter and simpler to read by just having `+ cause` at the end.

Reply:
Fair point, I had filename there originally and it evolved, but i didn't remove the format.